tests/Client/AzureIsvCommandTests.cs

```diff
@@ -0,0 +1,35 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+using System.Text.Json;
+using AzureMcp.Tests.Client.Helpers;
+using Xunit;
+
+namespace AzureMcp.Tests.Client;
+
+public class AzureIsvCommandTests(LiveTestFixture liveTestFixture, ITestOutputHelper output)
+    : CommandTestsBase(liveTestFixture, output),
+    IClassFixture<LiveTestFixture>
+{
+    [Fact]
+    [Trait("Category", "Live")]
+    public async Task Should_list_datadog_monitored_resources()
+    {
+        // Skipping test if Tenant is not 'Customer LED Tenant'
+        if (Settings.TenantId != "888d76fa-54b2-4ced-8ee5-aac1585adee7")
```

Comment:
Seems like the live test will not run in TME, either our playground.

Is this by design that we don't want to run the test in our regular CI?